modules/database/database.bal

```diff
@@ -732,13 +732,14 @@ public isolated transactional function getStepsUsingData(int|ExperimentDataFull
 # + 'version - The plugin version  
 # + status - Filter for the current status of the timeline step result
 # + uncleared\-substep - Filter for (un)cleared substeps. If set to 1 (or any postivie number), steps must have at least one uncleared substeps. Else, must be set to -1 (or any negative number). Set to 0 if not specified.
+# + result\-quality - Filter for the result quality of the timeline step result
```

Comment:
you only need the `-` in the query param. One level down in the code you can use camelCase again (which just looks better)